pycoal/tests/mineral_test.py

```diff
@@ -81,7 +87,34 @@ def test_classify_image():
         # verify that every pixel has the same classification
         assert numpy.array_equal(expected.asarray(), actual.asarray())
 
-
+# verify that classified images have valid classifications when using config file
+# three basic tests w/ diff parallel methods and loading image in mem
+
+# currently dask not supported in config file
+'''
+@with_setup(_init_config_parser, _test_classify_image_teardown)
+def test_classify_image_dask(config):
+    # set our config file parameter ['processing']['impl'] to 'dask'
+    config['processing']['impl'] = 'dask'
+    test_classify_image()
+    test_classify_image_in_memory()
+'''
+
+@with_setup(_init_config_parser, _test_classify_image_teardown)
+def test_classify_image_pytorch():
+    # set our config file parameter ['processing']['impl'] to pytorch
+    config['processing']['impl'] = 'pytorch'
+    test_classify_image()
+    test_classify_image_in_memory()
+
+@with_setup(_init_config_parser, _test_classify_image_teardown)
+def test_classify_image_joblib(config):
+    # set our config file parameter ['processing']['impl'] to joblib
+    config['processing']['impl'] = 'joblib'
+    test_classify_image()
+    test_classify_image_in_memory()
+
+    
```

Comment:
You should also introduce a test which tests for incorrect/wrong/dummy configuration. This tests the new config-loading logic you've implemented.